integrations/spark/spark-job-dataquality/spark-job-dataquality-app/src/main/java/com/thinkbiganalytics/spark/dataquality/rule/DataQualityRule.java

```diff
@@ -0,0 +1,75 @@
+package com.thinkbiganalytics.spark.dataquality.rule;
+
+/*-
+ * #%L
+ * kylo-spark-job-dataquality-app
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import com.thinkbiganalytics.spark.dataquality.util.FlowAttributes;
+import org.json.simple.JSONObject;
+
+/**
+ * Interface for Data Quality rules
+ */
+public interface DataQualityRule {
+
+    /**
+     * Sets the attributes based on the passed in parameter
```

Comment:
Either a . or <br> at the end of comment. Please check in all the files for similar issues where javadocs are not rendered properly, and update.